operator/src/main/java/oracle/kubernetes/operator/http/HttpResponseStep.java

```diff
@@ -21,7 +21,15 @@ public HttpResponseStep(Step next) {
 
   @Override
   public NextAction apply(Packet packet) {
-    return Optional.ofNullable(getResponse(packet)).map(r -> doApply(packet, r)).orElse(doNext(packet));
+    return Optional.ofNullable(getResponse(packet))
+        .map(r -> doApply(packet, r))
```

Comment:
This logic has gotten complex, with two separate `orElse` cases. Can the contents of the first one be extracted to a new method?